test/libprivate/include/SimplePrivate.h

```diff
@@ -0,0 +1,12 @@
+/*******************************************************************************
+ * Copyright (C) 2024 Advanced Micro Devices, Inc.
+ ******************************************************************************/
+
+
+#ifndef GUARD_SIMPLE_PRIVATE_H
+#define GUARD_SIMPLE_PRIVATE_H
+
+void simple-private();
```

Comment:
This should be `simple_private` as `-` is not valid symbol identifier.

Reply:
Thanks for catching that, I missed to change it here. Done.